### ISSUE TITLE
Undocked docking ports throw null reference

### ISSUE THREAD

**Issue by SumGuyTwitches** [SumGuyTwitches/FuelMyShip#1]

**Comment by linuxgurugamer** [comment]
Probably adding a check here:

https://github.com/SumGuyTwitches/FuelMyShip/blob/b48fd4edd19163bcf34b399bf33a3e48e6538e54/FuelMyShip/FuelMyShipModule.cs#L58

to only show it if both nodes are connected would fix this (just guessing right now)

**Comment by SumGuyTwitches** [comment]
@linuxgurugamer agreed, I'll update the mod to only show, or perhaps make active, the transfer button when docked, should have the secondary effect of not being available on "struts docks"
